### PR TITLE
expand description into different components

### PR DESCRIPTION
**Pull request by InbarDanieli**
I changed the description page into its own component that you can use in different pages and add different content depend on your needs in the page

### CODE REVIEW

File: components/description/Description.tsx [hunk 1]
```diff
@@ -2,26 +2,10 @@ import React from 'react';
 import style from './description.module.scss';
 import colors from '../../styles/colors';
 
-const Description = () => {
+const Description = (props) => {
```
Comment (MichalPorag) [inline components/description/Description.tsx:5]: Please spread props

File: components/description/Description.tsx [hunk 2]
```diff
@@ -2,26 +2,10 @@ import React from 'react';
...
-          <br /> בנוסף מוזמנים להצטרף להרצאות העשרה, לשתף שאלות ובקשות לעזרה ולשתף מהידע שלכם בקבוצה
-          שלנו בפייסבוק.
-          <br />
-          אם אתם רוצים להצטרף כמנטורים של פרויקט,
-          <br /> מוזמנים למלא את הטופס הבא:
-          <a href="https://forms.gle/6mHgTs5PnsfXUZ1f8">✍</a>
-        </p>
-      </div>
+      {props.descriptionOutput}
       <ul className={style.squareContainer}>
```
Comment (MichalPorag) [inline components/description/Description.tsx:9]: please remove squares

File: components/main-page-description/MainPageDescription.module.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.subDesc {
+  font-size: 3rem;
+  margin-top: 15px;
```
Comment (MichalPorag) [inline components/main-page-description/MainPageDescription.module.scss:3]: remove

File: components/main-page-description/MainPageDescription.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import colors from '../../styles/colors';
+import style from './MainPageDescription.module.scss';
+
+function MainPageDescription() {
+  return (
+    <div>
```
Comment (MichalPorag) [inline components/main-page-description/MainPageDescription.tsx:7]: please use flexbox and gap

Comment (MichalPorag) [reply components/main-page-description/MainPageDescription.tsx:7]: please make it fragment and move the flexbox logic to the parent component

File: components/main-page-description/MainPageDescription.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      </h3>
+      <p className={style.subDesc}>
+        מוזמנים לתרום לפרויקט של הקהילה או לפרויקטים שרצים תחתיה בסדנאות שרצות אחת לחודש לתרומה לקוד
+        פתוח.
+        <br /> בנוסף מוזמנים להצטרף להרצאות העשרה, לשתף שאלות ובקשות לעזרה ולשתף מהידע שלכם בקבוצה
+        שלנו בפייסבוק.
+        <br />
+        אם אתם רוצים להצטרף כמנטורים של פרויקט,
+        <br /> מוזמנים למלא את הטופס הבא:
+        <a href="https://forms.gle/6mHgTs5PnsfXUZ1f8">✍</a>
```
Comment (MichalPorag) [inline components/main-page-description/MainPageDescription.tsx:21]: please add alt

File: components/main-page-description/MainPageDescription.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import colors from '../../styles/colors';
+import style from './MainPageDescription.module.scss';
+
+function MainPageDescription() {
+  return (
+    <div>
+      <h3>
+        <span style={{ color: colors.secondary_color }}>Pull Request</span> עוזרת לכם להיכנס
```
Comment (MichalPorag) [inline components/main-page-description/MainPageDescription.tsx:9]: please don't use inline style

**Review by MichalPorag — CHANGES_REQUESTED** [review]
Look great! just a few changes please

File: components/main-page-description/MainPageDescription.module.scss (hunk 3 above)
Comment (InbarDanieli) [reply components/main-page-description/MainPageDescription.module.scss:3]: the code or the whole page?

File: components/main-page-description/MainPageDescription.tsx (hunk 5 above)
Comment (InbarDanieli) [reply components/main-page-description/MainPageDescription.tsx:21]: what do you mean by alt?
change it to an image?

File: components/description/Description.tsx (hunk 1 above)
Comment (InbarDanieli) [reply components/description/Description.tsx:5]: what do you mean by spread props?

Comment (MichalPorag) [reply components/description/Description.tsx:5]: change to ({descriptionOutput})

File: components/main-page-description/MainPageDescription.tsx (hunk 5 above)
Comment (MichalPorag) [inline components/main-page-description/MainPageDescription.tsx:21]: change it to open this in another tab

**Review by MichalPorag — CHANGES_REQUESTED** [review]

**Review by MichalPorag — APPROVED** [review]

**Pull request merged**